features/carts/firebase_cart_dao.py

```diff
@@ -1,68 +1,78 @@
+import asyncio
+import concurrent.futures
 from google.cloud import firestore
 
 from constants import CollectionName
 from helpers import Helpers
 from .cart_dao import CartDao
 
+executor = concurrent.futures.ThreadPoolExecutor(max_workers=20)
```

Comment:
You can use the default executor by passing `None` to wherever executor is expected. The default one is already a threadpool executor.

Reply:
Thanks